content/docs/capabilities/ppl.mdx

```diff
@@ -76,6 +76,41 @@ If `nor` is used, the user will have access if their email address does not end
   </div>
 </details>
 
+### Multiple Operators in an Action
+
+You _can_ add multiple operators under the same Action block, and it would be valid PPL.
+
+For example, this policy would grant access to anyone with an `example<X>.com` domain in their email address:
+
+```yaml
+allow:
+  and:
+    - domain:
+        is: example1.com
+  or:
+    - domain:
+        is: example2.com
+    - domain:
+        is: example3.com
+```
+
+However, you could write an identical policy with multiple `allow` blocks:
+
+```yaml
+allow:
+  and:
+    - domain:
+        is: example1.com
+allow:
+  or:
+    - domain:
+        is: example2.com
+    - domain:
+        is: example3.com
+```
+
+Although these policies are equally effective, the second example using multiple Actions is the intended method to express PPL.
```

Comment:
I'm not sure this makes for a good example. This policy could be further simplified using a single `or` operator, right?

```yaml
allow:
  or:
    - domain:
        is: example1.com
    - domain:
        is: example2.com
    - domain:
        is: example3.com
```